### PR TITLE
SW-4521 Implement Message component for reports that need to be completed

### PR DESCRIPTION
**Pull request by nickgraz**
Use `@terraware/web-components:2.4.1-rc.0` for the Message component `body` prop render update.

### CODE REVIEW

**Comment by nickgraz** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #2013** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2013?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/2013?utm_source=stack-comment).

File: src/components/Reports/ReportsView.tsx [hunk 1]
```diff
@@ -130,12 +131,60 @@ export default function ReportsView(props: ReportsViewProps): JSX.Element {
...
+    const reportPeriods = reportsToComplete.map((report) => `${report.year}-Q${report.quarter}`).sort();
+
+    const dedupedPeriods: string[] = [];
+    for (const period of reportPeriods) {
+      if (!dedupedPeriods.includes(period)) {
+        dedupedPeriods.push(period);
+      }
+    }
+
+    return dedupedPeriods.join(', ');
```
Comment (karthikbtf) [inline src/components/Reports/ReportsView.tsx:150]: suggestion if you prefer this
```
  return Array.from(new Set(reportsToComplete.map((report) => `${report.year}-Q${report.quarter}`))).sort();
```

**Review by karthikbtf — APPROVED** [review]
Nice!!

File: src/components/Reports/ReportsView.tsx [hunk 2]
```diff
@@ -138,16 +138,9 @@ export default function ReportsView(props: ReportsViewProps): JSX.Element {
...
-
-    const dedupedPeriods: string[] = [];
-    for (const period of reportPeriods) {
-      if (!dedupedPeriods.includes(period)) {
-        dedupedPeriods.push(period);
-      }
-    }
-
-    return dedupedPeriods.join(', ');
+    return Array.from(new Set(reportsToComplete.map((report) => `${report.year}-Q${report.quarter}`)))
```
Comment (karthikbtf) [inline src/components/Reports/ReportsView.tsx:141]: Thanks!

**Review by karthikbtf — APPROVED** [review]
🎉

**Pull request merged**